src/main/java/org/kohsuke/github/GHRelease.java

```diff
@@ -123,9 +123,14 @@ public GHAsset uploadAsset(File file, String contentType) throws IOException {
 
         String url = format("https://uploads.github.com%s/releases/%d/assets?name=%s",
                 owner.getApiTailUrl(""), getId(), file.getName());
-        return builder.contentType(contentType)
-                .with(new FileInputStream(file))
+        FileInputStream istream = new FileInputStream(file);
+        try {
+            return builder.contentType(contentType)
+                .with(istream)
```

Comment:
I would annotate `Requester.with()` like this:
```java
    public Requester with(@WillClose InputStream body) { /*etc*/}
```
The closing is already handled in `Requester`. :(